AnkiDroid/src/main/java/com/ichi2/anki/DeckPicker.java

```diff
@@ -1573,6 +1642,13 @@ public void sync() {
     @Override
     public void sync(Connection.ConflictResolution syncConflictResolution) {
         SharedPreferences preferences = AnkiDroidApp.getSharedPrefs(getBaseContext());
+
+        // Block sync if 'legacyPath' preference exists, i.e, user data is currently being migrated
```

Comment:
```suggestion
        // Block sync if migration source path preference exists, i.e, user data is currently being migrated
```

It would also be good if `migrationSourcePath` was a constant, just in case